src/core/cells.hpp

```diff
@@ -72,15 +72,18 @@
 /** Flag indicating that there is no cell system yet. Only at the
  * VERY beginning of the code startup.
  */
-#define CELL_STRUCTURE_NONEYET -1
-/** Flag indicating that the current cell structure will be used further on */
-#define CELL_STRUCTURE_CURRENT 0
-/** cell structure domain decomposition */
-#define CELL_STRUCTURE_DOMDEC 1
-/** cell structure n square */
-#define CELL_STRUCTURE_NSQUARE 2
-/** cell structure layered */
-#define CELL_STRUCTURE_LAYERED 3
+enum {
```

Comment:
Here the Doxygen syntax gets broken. The group description should become an enum description, e.g.:
```c++
/** Cell Structure */
enum {
  /** Flag indicating that there is no cell system yet. Only at the
   *  VERY beginning of the code startup.
   */
  CELL_STRUCTURE_NONEYET = -1,
  /** Flag indicating that the current cell structure will be used further on */
  CELL_STRUCTURE_CURRENT = 0,
  /** cell structure domain decomposition */
  CELL_STRUCTURE_DOMDEC = 1,
  /** cell structure n square */
  CELL_STRUCTURE_NSQUARE = 2,
  /** cell structure layered */
  CELL_STRUCTURE_LAYERED = 3
};
```